tests/data/nonce

```diff
@@ -0,0 +1 @@
+l�.��v�,��H
```

Comment:
can you add a readme in the `testa/data` file describing each file in this directory?
are public.pub / private.pem a keypair? what are they used for?

e.g. `signing_key.pub` is used to sign an enclave in which the pre-generated evidence `evidence/` was generated. `evidence` contains that enclave's public key `report_key.pub` and the nonce `nonce`. 

(i'm not sure if what i said is exactly correct, but hopefully conveys my idea)

Reply:
Good idea! Added